gradle.properties

```diff
@@ -1,10 +1,9 @@
 projectVersion=3.1.1-SNAPSHOT
```

Comment:
Can you bump the minor version since this change includes minor version bumps to the dependencies

Reply:
Done.